config/tomlConfig_test.go

```diff
@@ -839,10 +840,13 @@ func TestEnableEpochConfig(t *testing.T) {
 
     # CurrentRandomnessOnSortingEnableEpoch represents the epoch when the current randomness on sorting is enabled
     CurrentRandomnessOnSortingEnableEpoch = 93
-
+	
     # AlwaysMergeContextsInEEIEnableEpoch represents the epoch in which the EEI will always merge the contexts
     AlwaysMergeContextsInEEIEnableEpoch = 94
 
+	# CleanupAuctionOnLowWaitingListEnableEpoch represents the epoch when the cleanup auction on low waiting list is enabled
+	CleanupAuctionOnLowWaitingListEnableEpoch = 95
+
```

Comment:
Is this a github display issue or a format issue?

Reply:
tabs vs spaces - fixed